documentation/how-to/tracking.en.md

```diff
@@ -38,6 +38,9 @@ values entered in the form remembered across features.
 !!! note
     QField will skip the feature form step if the layer is configured to hide all attributes or if form has been set to hide on feature addition in the attribute form settings. This can streamline the process of starting tracking sessions.
 
+!!! note
+    QField offers the capability to resume tracking sessions seamlessly, even if the application has been restarted or the device has been rebooted. This ensures continuity by allowing previously started tracking sessions to be reactivated without loss progress.
+
 To prevent overlapping, you have the option to set a maximum distance tolerance between the last and next tracked point/vertices. This ensures that if the position exceeds this defined distance, it is considered as an erroneous entry.
```

Comment:
Not directly linked to your addition here, but let's move this last paragraph of the section to site _above_ the two notes, otherwise it looks like an orphan floating around :)

Reply:
Good point! I'll move the paragraph above the two notes. Thanks for the suggestion!